parser.go

```diff
@@ -107,31 +106,27 @@ func NewCheckStyleParser() Parser {
 	return &CheckStyleParser{}
 }
 
-func (p *CheckStyleParser) Parse(r io.Reader) ([]*CheckResult, error) {
+func (p *CheckStyleParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method CheckStyleParser.Parse should have comment or be unexported